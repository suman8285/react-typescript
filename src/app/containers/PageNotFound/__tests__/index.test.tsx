import React from 'react';
import { render } from '@testing-library/react';
import { PageNotFound } from '../index';
import { BrowserRouter } from 'react-router-dom';

describe('Page Not Found', () => {
  it('Matching the snapshot', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <PageNotFound />
      </BrowserRouter>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
