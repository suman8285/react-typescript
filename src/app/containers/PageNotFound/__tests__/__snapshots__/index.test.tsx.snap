// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found Matching the snapshot 1`] = `
<DocumentFragment>
  .c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 320px;
}

.c1 {
  margin-top: -8vh;
  font-weight: bold;
  font-size: 3.375rem;
}

.c1 span {
  font-size: 3.125rem;
}

.c2 {
  font-size: 16px;
  line-height: 1.5;
  margin: 10px 0 24px 0;
}

<div
    class="c0"
  >
    <div
      class="c1"
    >
      4
      <span
        aria-label="Crying Face"
        role="img"
      >
        ðŸ˜¢
      </span>
      4
    </div>
    <p
      class="c2"
    >
      Page not found.
    </p>
    <a
      href="/"
    >
      Return to Home Page
    </a>
  </div>
</DocumentFragment>
`;
